02-Logging/03-Saving-Array-Quantities.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>It seems like you're writing the tutorials within a group assuming that users are going in sequence (e.g. at the 02 ... index you list prereqs, but 02/03 assumes you've done 02/01 and 02/02). If that's the case, I don't think you need to copy paste all the explanations; if you feel it's helpful, maybe shorten it to a callback "as discussed in the previous section, we have to delete...".</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>